.github/workflows/upgrade-python-requirements.yml

```diff
@@ -48,7 +48,9 @@ jobs:
 
     steps:
       - name: setup target branch
-        run: echo "target_branch=$(if ['${{ inputs.branch }}' = '']; then echo 'master'; else echo '${{ inputs.branch }}'; fi)" >> $GITHUB_ENV
+        env:
+          BRANCH: ${{ inputs.branch }}
+        run: echo "target_branch=$(if ['$BRANCH' = '']; then echo 'master'; else echo '$BRANCH'; fi)" >> $GITHUB_ENV
```

Comment:
Not all repos use `master` as the default branch. You should use the default branch info - I think you can grab that in an action.

Reply:
quick google suggests `${{ github.event.repository.default_branch }}`